pay-queue/src/pay_queue/services/cgi_reconciliations.py

```diff
@@ -431,6 +434,22 @@ def _process_ap_header_routing_slips(line) -> bool:
     return has_errors
 
 
+def _process_ap_header_eft(line) -> bool:
+    has_errors = False
+    eft_refund_id = line[19:69].strip()
+    eft_refund: EFTRefundModel = EFTRefundModel.find_by_id(eft_refund_id)
```

Comment:
`eft_refund: EFTRefundModel = EFTRefundModel.find_by_id(eft_refund_id)`

type is already inferred

`eft_refund = EFTRefundModel.find_by_id(eft_refund_id)`